app/bundles/LeadBundle/Controller/AjaxController.php

```diff
@@ -809,27 +823,35 @@ protected function updateLeadFieldValuesAction(Request $request)
 
             $dataArray['fieldType'] = $leadFieldType;
             $dataArray['options']   = $options;
+        }
 
-            if ('field' === $changed) {
-                $dataArray['operators'] = $this->getModel('lead')->getOperatorsForFieldType($leadFieldType, ['date']);
-                foreach ($dataArray['operators'] as $value => $label) {
-                    $dataArray['operators'][$value] = $this->get('translator')->trans($label);
-                }
-
-                reset($dataArray['operators']);
-                $operator = key($dataArray['operators']);
-            }
-
-            $disabled = false;
-            switch ($operator) {
-                case 'empty':
-                case '!empty':
-                    $disabled = true;
-                    break;
+        if(in_array($alias, ['tags', 'segments'])){
+            $leadFieldType =      [
+                'include' => [
+                    '=',
+                    '!=',
+                    'regexp',
+                    '!regexp',
+                ],
+            ];
+        }
+        if ('field' === $changed) {
+            $dataArray['operators'] = $this->getModel('lead')->getOperatorsForFieldType($leadFieldType, ['date']);
+            foreach ($dataArray['operators'] as $value => $label) {
+                $dataArray['operators'][$value] = $this->get('translator')->trans($label);
```

Comment:
I applied CS Fixer and now works.
 I will apply it to my others PR :)
How Could I reproduce this notice error?